README.md

```diff
@@ -19,6 +20,32 @@ This webserver also exposes the follwing paths:
 - `/full_update`: send a `POST` here to run the full_update script. Note that
   this will not delete any files (yet).
 
-## Usage
+## Usage / Documentation
 
-See the [example folder](example/)
+Documentation is available on [deno.land](https://deno.land/x/ghact?doc).
+
+### Example Usage
+
+main.ts:
+
+```ts
+import { GHAct, type Config } from "."
+const config: Config = { ... };
```

Comment:
This allows the config in the server and the worker to be different, as this usually shouldn't be the case, I suggest havin a `config.ts` module used by both.